index.html

```diff
@@ -13,8 +13,8 @@
             <div class = "container">
                 <div class="inner">
                     <h1>Experience the rush today</h1>
-                    <p class="text">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore</p>
-                    <p class="text">magna aliqua consectetur adipisicing elit.</p>
+                    <p class="text">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore<br>

```

Comment:
`<br>` is not needed

Reply:
In this case the line may be wider than in psd. 
Should I set a width param for paragraph or fix it's width in some another way?

Reply:
Yeah the width of paragraph should be the same as in PSD